elastic/kibana
# [Discover] Validate if Data View time field exists on Alert creation / editing

## Summary

Closes #135806

This PR adds optional `timeField` param for Discover alert and adding validation data view if it time based.

![AD61D10F-6278-429C-B69D-C1952BB0A3C1_4_5005_c](https://user-images.githubusercontent.com/39378793/205312590-0392cd2e-740e-4e3e-ba17-712e0696eef3.jpeg)

### How to test
- Open `Alerts` in Discover
- Select non time based data view
- Try to save the rule. You should see error message.


### Checklist
